### DISCUSSION TITLE
how to set font for english and other characters separately in a cell?

### DISCUSSION THREAD

**Discussion by libinxu01** [lvjr/tabularray#460] [Q&A]
I need to set Times New Roman font for english characters and Source Han Sans SC Medium for other characters in a cell, how to achieve it?

**Comment by muzimuzhi** [comment]
If you can already achieve the same effect outside table cells, `cell{<row>}{<col>}={font=<setting>}` should work for a cell in `tabularray` table.  Otherwise, you're facing a general LaTeX font setting task, which is not related to `tabularray` hence is recommended to be asked on general Q&A forum, like tex.stackexchange.com.

**Reply by libinxu01** [reply]
Get it. Thanks a lot!
